packages/certora/specs/Claimer.spec

```diff
@@ -1,8 +1,32 @@
+using SymbolicSmartVault as ssv;
 
-rule sanity() {
+methods {
+    function _.balanceOf(address) external          => DISPATCHER(true);
+    function _.allowance(address,address) external  => DISPATCHER(true);
+    function _.approve(address,uint256) external    => DISPATCHER(true);
+    function _.transfer(address,uint256) external   => DISPATCHER(true);
+    function _.transferFrom(address,address,uint256) external => DISPATCHER(true);
+    
+    function SymbolicSmartVault.claimingToken() external returns (address) envfree;
+    function SymbolicSmartVault.claimingAmount() external returns (uint256) envfree;
+
+    function getClaimableBalance(address) external returns (uint256) envfree;
+    function _buildData(address token, uint256 amount) internal returns (bytes memory) => NONDET;
```

Comment:
You're not actually using that function right?
You ended up moving this call to the smart vault?